packages/perseus/src/widgets/explanation/explanation.test.ts

```diff
@@ -302,10 +302,7 @@ describe("Explanation", function () {
 
     describe("static validate", () => {
         it("should always return 0 points", async () => {
-            const result = ExplanationWidgetExports.widget.validate(
-                {},
-                question1.widgets["explanation 1"].options,
-            );
+            const result = ExplanationWidgetExports.widget.validate();
```

Comment:
Question: Should this be a call to the validator function instead? I know it eventually gets to it as it goes through `validate()`, but it sounded like we wanted to replace all of those with the validator function.

Reply:
My $0.02 worth. :) I think leaving it like this for now is good. Keeps our changes more incremental. We'll do another round of separating the React components completely from validation.